packages/smart/contracts/libraries/Rewardable.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+abstract contract Rewardable {
+    function getRewardEndTime(uint256 _eventId) public virtual returns (uint256);
```

Comment:
Not everything uses events. Probably should be generic like `uint256 _id`, with a comment explaining that it could refer to a market OR some grouping of markets, such as an event or group.